python/cudf/cudf/core/dataframe.py

```diff
@@ -2551,6 +2555,16 @@ def insert(self, loc, name, value, nan_as_null=None):
             name or label of column to be inserted
         value : Series or array-like
         """
```

Comment:
Unrelated, but I notice that `nan_as_null` is not documented and it should be.

Reply:
Added docs